package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "test": "mocha -r ts-node/register test/libraryUtilitiesTest.ts",
```

Comment:
@fanwgwg do you think we can merge `test`, `testjest` and `testcover` into one command `test`? Either case, we need `ReadMe.md` updated.